fastchat/serve/gradio_block_arena_anony.py

```diff
@@ -191,52 +200,113 @@ def share_click(state0, state1, model_selector0, model_selector1, request: gr.Re
     "dolly-v2-12b": 0.1,
     "llama-13b": 0.1,
     "chatglm-6b": 0.5,
+    "deluxe-chat-v1": 4,
 }
 
-SAMPLING_BOOST_MODELS = []
+SAMPLING_BOOST_MODELS = ["openchat-3.5", "gpt-4-turbo", "gpt-3.5-turbo-1106"]
+OUTAGE_MODELS = [
+    "claude-1",
+    "claude-2",
+    "claude-instant-1",
+    "zephyr-7b-alpha",
+    "wizardlm-70b",
+    "falcon-180b-chat",
+]
+
+
+def get_sample_weight(model):
+    if model in OUTAGE_MODELS:
+        return 0
+    weight = SAMPLING_WEIGHTS.get(model, 1.0)
+    if model in SAMPLING_BOOST_MODELS:
+        weight *= 5
+    return weight
+
+
+def get_battle_pair():
+    if len(models) == 1:
+        return models[0], models[0]
+
+    targets = {
```

Comment:
To keep consistent, can we move this `targets` as a global constant after `SAMPLING_BOOST_MODELS` ?

Reply:
moved. take a look?